site/docs/.vuepress/configs/uk.ts

```diff
@@ -176,7 +176,7 @@ export const localeUk: LocaleConfig<DefaultThemeLocaleData> = {
                 link: "/uk/plugins/menu.html",
               },
               {
-                text: "Питання без стану (stateless-question)",
+                text: "Розмови без стану (stateless-question)",
```

Comment:
```suggestion
                text: "Питання без стану (stateless-question)",
```
Думаю, краще залишити "питання". Це більш відповідає призначенню плагіна, а також використання "розмов" створить плутанину з плагіном для розмов.